docs/tre-developers/resource-processor.md

```diff
@@ -18,50 +18,37 @@ Resource Processor is the Azure TRE component automating [Porter](https://porter
 
 ## Local development
 
-To work locally checkout the source code and run:
+To work locally, checkout the source code and run:
```

Comment:
how does this fit in with @ross-p-smith's work for setting up local debugging? do we want a `make setup-local-debugging-rp` target instead of the manual steps?

Reply:
That's the plan, didn't want to spend too much time adding this before getting the PR sorted as its not part of the work item, just something I wanted to make easier for my own purposes and thought it was worth sharing

Reply:
Will likely add it as part of my next PR or spin up a dedicated task for it